packages/cadl-ranch-specs/http/projection/main.cadl

```diff
@@ -0,0 +1,59 @@
+import "@cadl-lang/rest";
+import "@azure-tools/cadl-ranch-expect";
+
+using Cadl.Http;
+
+@doc("Projection")
+@supportedBy("dpg")
+@scenarioService("/projection")
+namespace Projection;
```

Comment:
Think this should either be `ProjectedName` or `TargetProjection` to describe it is testing just that one `target` projection